lib/job-simulation.rb

```diff
@@ -1,31 +1,58 @@
-require './Stack.rb'
-require './Queue.rb'
+require_relative 'Stack'
+require_relative 'Queue'
 
 class JobSimulation
-  attr_reader :workers, :waiting, :roll
+  attr_reader :employed, :waiting, :waitlist
```

Comment:
You shouldn't change the variables that I put here. If this is the spec that's given to you then these are the names of the variables that will be expected in other places, so if you change it here, other programs expecting these names won't work properly. Also no adding variables (`waitlist`) or removing variables (`roll`)